.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks/5
+    url: https://community.auth0.com
     about: Discuss this SDK in the Auth0 Community forums
-  - name: Library Documentation
-    url: https://github.com/auth0/passport-auth0#documentation
-    about: Read the library docs on Auth0.com
```

Comment:
There is only one link there, and it points to the Auth0 docs site. It does not contain any docs for this library in particular, so it can be confusing and frustrating.

<img width="522" alt="Screenshot 2023-04-26 at 8 39 08 PM" src="https://user-images.githubusercontent.com/5055789/234725147-e8aa9c1c-8a7e-4528-9a63-f980ab033a64.png">